code/modules/surgery/implant.dm

```diff
@@ -137,6 +137,8 @@
 		affected.owner.custom_pain("You feel something rip in your [affected.name]!", 1)
 	user.drop_item()
 	affected.implants += tool
+	if(istype(tool, /obj/item/device/gps))
+		moved_event.register(target, tool, /obj/item/device/gps/proc/update_position)
```

Comment:
You will need to unregister this event, if the GPS is removed / destroyed / whatever.

Reply:
I did wonder about that, but it seemed to stop updating the former implant target's location once removed. 

I have explicitly unregistered it when the implant is removed, but now I'm curious about gps/Destroy() when implanted. Will it need to have a different unregister call when implanted? Will GPS need an implanted_into var or similar that's set when implanting?

Can easily do, just want to make sure I'm going about it the right way.

Reply:
I've also done the above now.

Reply:
Looks good.